packages/terra-brand-footer/tests/jest/BrandFooter.test.jsx

```diff
@@ -59,10 +122,14 @@ describe('BrandFooter', () => {
   it('should render with provided links and content left', () => {
```

Comment:
IMO we should keep these tests that are verify the links prop works and write new tests for the sections prop. Although you have marked `links` as deprecated, we still want to test and ensure both props render as expected until `links` is removed.

Reply:
Done in 0939f42333450ab5a565782924d03fc9904b8564